subprojects/core-api/src/main/java/org/gradle/api/provider/HasMultipleValues.java

```diff
@@ -152,4 +133,15 @@ public interface HasMultipleValues<T> extends HasConfigurableValue {
      */
     @Override
     void finalizeValue();
+
+    /**
+     * Updates the value of this property in place by executing the provided transformer.
+     * The transformer accepts the frozen value of this property.
```

Comment:
Is "frozen" the same thing as finalized?

Reply:
Also from @mlopatkin's PR, also explained here: https://github.com/gradle/gradle/pull/26564#discussion_r1375583508